src/widgets/buttons.ts

```diff
@@ -413,7 +435,7 @@ export function setImage (element: HTMLElement, thing: NamedNode) { // 20191230a
 // See, e.g., http://stackoverflow.com/questions/980855/inputting-a-default-image
 export function faviconOrDefault (dom: HTMLDocument, x: NamedNode) {
   const image = dom.createElement('img')
-  ;(image as any).style = style.iconStyle
+  ;(image as any).style = iconStyle
```

Comment:
What's with the semicolon at the beginning of this line?

Reply:
Because of the parentheses then semicolon has to be there to seperate the statement from the above line. Personally I would put it on the end of the line above, but I am not sure whether a different style is enforced on solid-ui